tests/system/aiplatform/test_e2e_tabular.py

```diff
@@ -119,21 +122,31 @@ def test_end_to_end_tabular(self, shared_state):
             create_request_timeout=None,
         )
 
-        automl_model = automl_job.run(
+        automl_job.run(
             dataset=ds,
             target_column="median_house_value",
             model_display_name=self._make_display_name("automl-housing-model"),
             sync=False,
         )
 
-        shared_state["resources"].extend(
-            [automl_job, automl_model, custom_job, custom_model]
-        )
+        while (
+            automl_job.state != gca_pipeline_state.PipelineState.PIPELINE_STATE_RUNNING
+        ):
+            time.sleep(5)
+
+        # Cancel the AutoML job once it's successfully been created, this is async
+        automl_job.cancel()
+
+        shared_state["resources"].extend([automl_job, custom_job, custom_model])
 
-        # Deploy both models after training completes
+        # Deploy the custom model after training completes
         custom_endpoint = custom_model.deploy(machine_type="n1-standard-4", sync=False)
-        automl_endpoint = automl_model.deploy(machine_type="n1-standard-4", sync=False)
```

Comment:
Preference to deploy and undeploy the model from the persisted endpoint instead of keeping a long-lived model deployment.

Reply:
Updated to deploy and undeploy a temporary endpoint.